tests/Unit.Tests/Services/GoogleAnalytics.Tests.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Net.Http;
+using System.Web;
+using FluentAssertions;
+using GovUk.Education.SearchAndCompare.Domain.Client;
+using GovUk.Education.SearchAndCompare.Services;
+using Moq;
+using NUnit.Framework;
+
+namespace GovUk.Education.SearchAndCompare.UI.Unit.Tests.Services
+{
+    class GoogleAnalyticsTests
+    {
+        [Test]
+        public void ProperlyFormedRequest()
+        {
+            var mockHttp = new Mock<IHttpClient>();
+            Uri resUri = null;
+            StringContent resContent = null;
+
+            mockHttp.Setup(x => x.PostAsync(It.IsAny<Uri>(), It.IsAny<StringContent>()))
+                .Callback((Uri a, StringContent b) => {resUri = a; resContent = b;})
+                .ReturnsAsync(new HttpResponseMessage());
+
+            var sut = new GoogleAnalyticsClient(mockHttp.Object, "abc");
```

Comment:
Me no likey `sut` as a name, I have to use my brain more to read the code. Suggest `googleAnalyticsClient` which describes what the variable contains.